code/game/gamemodes/antag_spawner.dm

```diff
@@ -135,35 +150,59 @@
 	if(!(user.mind in ticker.mode.syndicates))
 		user << "<span class='danger'>AUTHENTICATION FAILURE. ACCESS DENIED.</span>"
 		return 0
-	borg_to_spawn = input("What type?", "Cyborg Type", type) as null|anything in possible_types
-	if(!borg_to_spawn)
-		return
+
+	if(spawn_type == BORG_TELE)
+		borg_to_spawn = input("What type?", "Cyborg Type", type) as null|anything in possible_types
+		if(!borg_to_spawn)
+			return
 	var/list/borg_candicates = get_candidates(ROLE_OPERATIVE, 3000, "operative")
 	if(borg_candicates.len > 0)
 		used = 1
 		var/client/C = pick(borg_candicates)
 		spawn_antag(C, get_turf(src.loc), "syndieborg")
+		var/datum/effect_system/spark_spread/S = new /datum/effect_system/spark_spread
+		S.set_up(4, 1, src)
+		S.start()
 	else
 		user << "<span class='warning'>Unable to connect to Syndicate command. Please wait and try again later or use the teleporter on your uplink to get your points refunded.</span>"
 
 /obj/item/weapon/antag_spawner/borg_tele/spawn_antag(client/C, turf/T, type = "")
-	if(!borg_to_spawn) //If there's no type at all, let it still be used but don't do anything
-		used = 0
-		return
-	var/datum/effect_system/spark_spread/S = new /datum/effect_system/spark_spread
-	S.set_up(4, 1, src)
-	S.start()
-	var/mob/living/silicon/robot/R
-	switch(borg_to_spawn)
-		if("Medical")
-			R = new /mob/living/silicon/robot/syndicate/medical(T)
-		else
-			R = new /mob/living/silicon/robot/syndicate(T) //Assault borg by default
-	R.key = C.key
-	ticker.mode.syndicates += R.mind
-	ticker.mode.update_synd_icons_added(R.mind)
-	R.mind.special_role = "syndicate"
-	R.faction = list("syndicate")
+	switch(spawn_type)
+		if(BORG_TELE)
+			var/mob/living/silicon/robot/R
+			switch(borg_to_spawn)
+				if("Medical")
+					R = new /mob/living/silicon/robot/syndicate/medical(T)
+				else
+					R = new /mob/living/silicon/robot/syndicate(T) //Assault borg by default
+			R.key = C.key
+			ticker.mode.syndicates += R.mind
+			ticker.mode.update_synd_icons_added(R.mind)
+			R.mind.special_role = "syndicate"
+			R.faction = list("syndicate")
+		if(HUMAN_TELE)
+			var/nuke_code = "Ask your leader!"
+			var/mob/living/carbon/human/M = new/mob/living/carbon/human(T)
+			C.prefs.copy_to(M)
+			M.key = C.key
+			var/obj/machinery/nuclearbomb/nuke = locate("syndienuke") in nuke_list
+			if(nuke)
+				nuke.r_code = nuke_code
+			M.mind.make_Nuke(T, nuke_code, 0, FALSE)
+
+
+/obj/item/weapon/antag_spawner/borg_tele/operative

```

Comment:
Because it'd be a massive amount of copypaste with all the checks/messages and then someone would yell at me